sonar-project.properties

```diff
@@ -6,7 +6,6 @@ sonar.projectKey=dgrsc
 
```

Comment:
Just realized this should be `document-generation-showcase` but this is minor and can be updated on next PR.

Reply:
that value is never used.  it's always set in the Jenkinsfile